libs/coin-modules/coin-aptos/src/logic/lastBlock.ts

```diff
@@ -0,0 +1,7 @@
+import type { BlockInfo } from "@ledgerhq/coin-framework/api/index";
+import { AptosAPI } from "../network";
+
+export async function lastBlock(): Promise<BlockInfo> {
+  const client = new AptosAPI("aptos");
```

Comment:
Could we keep a single client across the integration ?

Reply:
yes, I'll make the changes for that